### ISSUE TITLE
Pager not working correctly after faster switching slides

### ISSUE THREAD

**Issue by erikgyepes** [malsup/cycle#100]
I have a problem with pager. When I am faster clicking on the next/prev button the pager starts to have a weird behavior. It is not assigning the active class correctly, sometimes it assigns to the next slides, then quickly switch back and so on. 

Anyone else has seen this?

**Comment by willmacdonald** [comment]
yes I am seeing this problem too, not found any solution for this.

**Comment by ericprice** [comment]
This also happens when activating different pager items too quickly. This bug was introduced sometime after v2.9995, as that version seems to work fine. Can't really tell what's causing it.

**Comment by 20pictures** [comment]
Is this similar to Issue #106 'Timeout speed up caused by hovering pagers' ?

**Comment by ericprice** [comment]
Yeah, #106 seems to be the same issue I'm having here.

**Comment by 20pictures** [comment]
I just spent 7 days staring at this issue & the only solution seems to be to trigger a page refresh or reload the Cycle. In my case the page refresh is not an option but the reloaded Cycle stalls on the 2nd slide...

**Comment by 20pictures** [comment]
I've just tried the older Version 2.9999.1 - before the bug fix where destroy did not unbind hover events ('pause' option)

It does seem slightly better than the new Version: 2.9999.2 (16-FEB-2012) also, make sure you use 
pagerEvent: 'mouseenter' NOT  pagerEvent: 'mouseover'

CORRECTION: The bug is the same in both versions.

**Comment by 20pictures** [comment]
Version: 2.9999.3 (08-MAR-2012) works for me!
